packages/kmod-6.1-nvidia-r570/kmod-6.1-nvidia-r570.spec

```diff
@@ -183,6 +188,11 @@ pushd NVIDIA-Linux-%{_cross_arch}-%{tesla_ver}/grid
 # compile the kernel, if we don't set this flag the compilation fails
 make %{?_smp_mflags} ARCH=%{_cross_karch} IGNORE_CC_MISMATCH=1 GRID_BUILD=1 GRID_BUILD_CSP=1 SYSSRC=%{kernel_sources} CC=%{_cross_target}-gcc LD=%{_cross_target}-ld
 
+# Strip symbols out of the .ko files
+for module in nvidia nvidia-uvm nvidia-modeset nvidia-drm nvidia-peermem; do
```

Comment:
I have the same concern as @rpkelly - why list specific kmods and run the risk that a new one will be added and this list won't be updated?

Reply:
You did this differently in the 6.12 version of the r570 spec. Please use `git diff --no-index` on the two package directories and eliminate any differences that aren't strictly necessary.